### ISSUE TITLE
Out Order of Elements Causes error 

### ISSUE THREAD

**Issue by ChrisAlvares** [ChrisAlvares/node-adwords#20]
Having elements outside of the normal adwords order causes issues with the sdk / api.

Test Case
```js
//this will work
let operation = {
    operator: 'ADD',
    operand: {
     ....
    }
}
```

```js
//this will not work
let operation = {
    operand: {
     ....
    },
    operator: 'ADD',
}
```

We might need to read the wsdl to determine the order of elements or something. Not really sure how to solve this one as dynamically as possible.

**Comment by talha-asad** [comment]
Yes this is an issue with XML soap requests, order needs to be preserved. I implemented this by reading the WSDL. It's done in my fork of the googleads-node-lib, `adwords-api`.

I really like the simplicity of this package though and thinking about deprecating that package as it's just too much mess with Backbone models etc.

**Comment by ChrisAlvares** [comment]
I will put it in the readme, but I think reading the wsdl might be a good thing. Perhaps even caching it in the package.

You don't necessarily need to read the WSDL as the documentation has everything in the correct order as well.
